src/components/NewWorkspaceModal.js

```diff
@@ -160,7 +163,21 @@ const NewWorkspaceModal = Utils.withDisplayName('NewWorkspaceModal', ({
           placeholder: 'Select a billing project',
           value: namespace,
           onChange: ({ value }) => setNamespace(value),
-          options: _.uniq(_.map('projectName', billingProjects)).sort()
+          styles: {
+            option: provided => ({
+              ...provided,
+              padding: 0
+            })
+          },
+          options: _.uniq(_.map(({ projectName, invalidBillingAccount }) => {
+            return {
+              label: h(Fragment, [
+                invalidBillingAccount ? h(TooltipTrigger, { content: ['Workspaces may only be created in billing projects that have a Google billing account accessible in Terra'], side: 'left' }, [div({ style: { padding: optionPadding } }, [projectName])]) : div({ style: { padding: optionPadding } }, [projectName])
+              ]),
+              value: projectName,
+              isDisabled: invalidBillingAccount
+            }
+          }, billingProjects)).sort()
```

Comment:
Couldn't insert it as a suggestion because I couldn't select the entire block due to the comments above but when I remove the optional `return` piece along with enclosing braces, and run the IntelliJ reformatter, I get the following:

```javascript
          options: _.uniq(_.map(({ projectName, invalidBillingAccount }) => ({
            label: h(Fragment, [
              invalidBillingAccount ?
                h(TooltipTrigger, {
                  content: ['Workspaces may only be created in billing projects that have a Google billing account accessible in Terra'], side: 'left'
                }, [div({ style: { padding: optionPadding } }, [projectName])]) :
                div({ style: { padding: optionPadding } }, [projectName])
            ]),
            value: projectName,
            isDisabled: invalidBillingAccount
          }),
          billingProjects)).sort()
```

It might be a bit easier to read that way, and the splitting the ternary into separate lines would be consistent with the rest of the codebase.